packages/core/test/anyOf_test.js

```diff
@@ -639,6 +678,52 @@ describe("anyOf", () => {
     });
   });
 
+  it("should use title from refs schema before using fallback generated value as title", () => {
```

Comment:
Can we just add one more test that includes a ref that refers to something within properties like (`{"$ref": "#/properties/test"}`) so we can ensure that functionality doesn't regress in the future?

Reply:
@mr-bjerre